src/app/EventManagement.cpp

```diff
@@ -413,6 +413,7 @@ void EventManagement::VendEventNumber()
 CHIP_ERROR EventManagement::LogEvent(EventLoggingDelegate * apDelegate, const EventOptions & aEventOptions,
                                      EventNumber & aEventNumber)
 {
+    assertChipStackLockedByCurrentThread();
```

Comment:
seems ok, however I still wonder if LogEventPrivate would be a better placement (to capture any future calls).

Adding checkmark since this seems to be the only method calling the private bit...

Reply:
yep, that is only public method that could be called in EventManagement.